README.md

```diff
@@ -1,39 +1,98 @@
-<p align="center">
-  <b>ConnectALL</b><br>
-  <i>Platform to help close the Digital Divide gap in collaboration with the White House</i><br>
-  <a href="http://make.sc/whitehouse">The Initiative</a> |
-  <a href="https://www.makeschool.com/founding-class">Make School</a>
-  <!--<a href="#">Press Kit</a>-->
-  <br><br>
-  <img src="https://www.whitehouse.gov/profiles/forall/modules/custom/gov_whitehouse_www/images/icons/wh_logo_seal.png" height="100" />
-</p>
+#[Go Digital with Us](http://www.godigitalwith.us/)
+![ruby version](https://img.shields.io/badge/ruby-2.3.1-brightgreen.svg)
+
+<center>
```

Comment:
Man, so hacky!  Looks like there are a couple of ways people have gotten around it: http://stackoverflow.com/questions/14051715/markdown-native-text-alignment.